src/angular.prefix

```diff
@@ -3,4 +3,4 @@
  * (c) 2010-2016 Google, Inc. http://angularjs.org
  * License: MIT
  */
-(function(window, document, undefined) {
+(function(window) {
```

Comment:
> It saves us from using `void 0` everywhere, no?

I'd just rely on the `undefined` from the outer scope. Note that it's not even possible in ES5 to overwrite `undefined`, it's immutable even in sloppy mode (the only difference is that in strict mode trying to overwrite `undefined` throws while in sloppy mode it's just a noop). The only way this could backfire is if Angular was loaded not via a regular script tag but wrapped in a separate closure which shadowed `undefined`. But that requires modifying the source and if someone does that we've already lost the battle.

> I agree it doesn't save us from the other stuff, but we do have various bits of defensive code in place in the case that people do override parts of the Object.prototype.

Since the global `undefined` is immutable if Angular is loaded normally so that its `undefined` points to the global one no one is able to modify that; shadowing is also impossible after the fact.

> What is the benefit in removing it?

There is no huge benefit, perhaps a tiny size decrease. If you feel strongly about it, I'll restore it. :) I just think it doesn't really solve any problem.